MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Defaults.xaml

```diff
@@ -40,9 +42,6 @@
         <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Thumb.xaml" />
         <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.ValidationErrorTemplate.xaml" />
         <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Window.xaml" />
-
-        <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Clock.xaml" />
```

Comment:
I thought the clock resource dictionary was originally singled out here because it's a custom control in this project, not one of WPF's built-in controls? 😀

Reply:
Actually I think that was a mistake that I never caught until reviewing this :)

Reply:
Also thank you for the contributions. It is really appreciated!